src/Queue/PayloadDeserializer.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Spiral\RoadRunnerBridge\Queue;
+
+use Spiral\Queue\HandlerInterface;
+use Spiral\Queue\HandlerRegistryInterface;
+use Spiral\Queue\SerializerRegistryInterface;
+use Spiral\RoadRunner\Jobs\Exception\JobsException;
+use Spiral\RoadRunner\Jobs\Task\ReceivedTaskInterface;
+
+final class PayloadDeserializer
```

Comment:
Might be a tokenizer listener to detect job handler payload type on bootstrapping